src/components/InputsBox.jsx

```diff
@@ -1,29 +1,35 @@
 import { Button, Grid, TextField, Box } from '@mui/material';
-import TextareaAutosize from '@mui/material/TextareaAutosize';
-import { useEffect, useState } from 'react';
+
+import { useState } from 'react';
 
 export default function InputsBox() {
-  const [selectedImage, setSelectedImage] = useState(null);
-  const [imageUrl, setImageUrl] = useState(null)
+    const [imageUrl, setImageUrl] = useState(null)
 
-  useEffect(() => {
-    if (selectedImage) {
-      setImageUrl(URL.createObjectURL(selectedImage));
+  function onImageChange(file) {
+    if(file && file['type'].split('/')[0] === 'image'){
+      setImageUrl(URL.createObjectURL(file))
+    } else {
+      alert ('This is not image file');
```

Comment:
Read how to provide error messages on a particular field in MUI
https://mui.com/components/text-fields/#validation